src/node/delay.rs

```diff
@@ -267,8 +264,8 @@ struct DelayReader {
     last_written_index: Rc<Cell<Option<usize>>>,
     // local copy of shared `last_written_index` so as to avoid render ordering issues
     last_written_index_checked: Option<usize>,
-    // internal buffer used to compute output per channel at each frame
-    internal_buffer: Vec<f32>,
+    // internal buffer used to compute output delay infos for each sample
+    playback_infos: [(usize, usize, usize, usize, f32); RENDER_QUANTUM_SIZE],
```

Comment:
It may be nice to introduce a struct for this, for readability
```
struct PlaybackInfo {
                prev_block_index: usize,
                prev_frame_index: usize,
                next_block_index: usize,
                next_frame_index: usize,
                k: f32,
}
```

Reply:
hum lost my comment...
done!